Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,179 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
```

Comment:
Try not to include week1 homework changes when submitting week2 homework.

You can do this by checkout out to the master `git checkout master` and creating a new branch from the master `git checkout -b week2-homework` that branches off of master, instead of creating a new branch that branches off of your other homework, which is what I think happened here.